.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish
```

Comment:
Will this workflow run even if the check workflow fails?

Reply:
Yes, they'll run in parallel. Arguably the action of tagging a commit already denotes a release whether it's produced an artifact in CI or not.